frontend/src/metabase/public/containers/PublicDashboard/PublicDashboard.tsx

```diff
@@ -126,14 +126,10 @@ class PublicDashboardInner extends Component<PublicDashboardProps> {
     }
 
     try {
-      const requests = [];
+      await fetchDashboardCardMetadata();
```

Comment:
It's a noop for public dashboards except "metadata loaded" flag; so it's safe not to call it in parallel